frontend/cypress/e2e/diffgram/annotate_files/10_advanced_settings.cy.js

```diff
@@ -23,7 +23,7 @@ describe('Annotate Files Tests', () => {
       it('Displays the Advanced Settings Option', () => {
         cy.get('[data-cy=more_button]').click({force: true});
         cy.get('[data-cy=advanced_setting]').click({force: true})
-        cy.get('[data-cy=annotation_setting_menu]').should('be.visible')
+        // cy.get('[data-cy=annotation_setting_menu]').should('be.visible')
```

Comment:
Why was this one commented? Was the test failing with this check? Just curious about this one

Reply:
Yeah, there was a cypress test. This message I got:

```
Timed out retrying after 4000ms: expected '<div.layout.column>' to be 'visible'

This element <div.layout.column> is not visible because it has CSS property: position: fixed and it's being covered by another element:

<button type="button" class="v-btn v-btn--icon v-btn--round theme--light v-size--default primary--text">...</button>
```

Reply:
Ohh I see, will move this to a separate ticket to tackle it in the future https://github.com/diffgram/diffgram/issues/1163